### ISSUE TITLE
count not work in processing.py mode

### ISSUE THREAD

**Issue by yonghuming** [processing/processing-sound#10]
```
add_library('sound')
def setup():
    n = WhiteNoise(this)
    n.play()
```

**Comment by yonghuming** [comment]
nothing happened

**Comment by yonghuming** [comment]
```

import processing.sound.*; WhiteNoise noise;  void setup() {   size(640, 360);   background(255);      // Create the noise generator   noise = new WhiteNoise(this);   noise.play(); }        void draw() { }
--


```
![image](https://user-images.githubusercontent.com/435079/45337149-37876f00-b5b9-11e8-9842-743a058d1b7f.png)

**Comment by kevinstadler** [comment]
@jdf can you help out here? I don't have processing.py set up so I can't reproduce

**Comment by kevinstadler** [comment]
Seems to be a more general issue with external libraries in Processing.py. More information is available at #23, so closing this issue.
